### PR TITLE
Fixed spelling mistake

### PR DESCRIPTION
**Pull request by edcs**

### CODE REVIEW

File: app/Http/Middleware/Authenticate.php [hunk 1]
```diff
@@ -37,6 +37,6 @@ public function handle($request, Closure $next)
             return $next($request);
         }
 
-        return redirect()->guest(route('oauth.authorise'));
+        return redirect()->guest(route('oauth.authorize'));
```
Comment (mikefrancis) [inline app/Http/Middleware/Authenticate.php]: omg m8 love it

**Comment by mikefrancis** [comment]
:+1: :information_desk_person: :nail_care:

File: app/Http/Middleware/Authenticate.php (hunk 1 above)
Comment (juanserra) [inline app/Http/Middleware/Authenticate.php]: Why not name it 'auth'?

Comment (edcs) [inline app/Http/Middleware/Authenticate.php]: Because we're using OAuth 2.0!

**Pull request merged**